src/ScheduleCommand.js

```diff
@@ -0,0 +1,22 @@
+import './Command'
+
+export class ScheduleCommand extends Command {
+
+	name = 'schedule:run'
+	description = 'Starts the schedule daemon and executes commands as scheduled.'
+
+	async run() {
+		this.info('Schedule daemon starting...')
+
+		const scheduled = []
+
+		for(let command of this.cli.commands) {
```

Comment:
`command.schedule` should also be a function now that it’s been added to the base Command class, so this can be simplified to:

```js
const scheduled = this.cli.commands.map(command => command.registerSchedule())
```